src/core/powerflow_data_wrapper.jl

```diff
@@ -0,0 +1,13 @@
+mutable struct PowerFlowEvaluationData{T <: PFS.PowerFlowData}
```

Comment:
@daniel-thom I added this new struct to store other information required to inter operate the PFData object the optimization container.